<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chartsOption</class>
 <widget class="QDialog" name="chartsOption">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,2,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="date1L">
         <property name="text">
          <string>Od:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="beginD">
         <property name="dateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2023</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="date2L">
         <property name="text">
          <string>Do:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="endD">
         <property name="dateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2023</year>
           <month>1</month>
           <day>2</day>
          </datetime>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="Nazwa_cwiczenia_L">
       <property name="text">
        <string>Nazwa ćwiczenia.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="Name_CB"/>
     </item>
     <item>
      <widget class="QLabel" name="GrupaC_L">
       <property name="text">
        <string>Grupa ćwiczenia</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="GroupEx_CB">
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
     <item>
      <widget class="QPushButton" name="chartB">
       <property name="text">
        <string>Wyświetl wykres dla wybranego ćwiczenia</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ranking">
       <property name="text">
        <string>Wyświetl ranking</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Wyjście</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
